e2e/scientists_queries.yaml

```diff
@@ -77,6 +77,22 @@ queries:
           # -num_rows: 5295 # greater than current limit
           - selected: ["?count", "?place"]
           - contains_row: [280, "<New_York_City>"]
+  - query: scientists-order-by-aggregate
+    solutions:
+      - type: no-text
+        sparql: |
+          SELECT ?place WHERE {
+              ?x <is-a> <Scientist> .
+              ?x <Place_of_birth> ?place .
+          }
+          GROUP BY ?place
+          ORDER BY DESC((COUNT(?x) as ?count))
+          LIMIT 1
+        checks:
+          - num_cols: 1
+          # -num_rows: 5295 # greater than current limit
+          - selected: ["?place"]
+          - res: [["<United_States_of_America>"]] 
```

Comment:
😂 nice trick to test for ordering, I think we really should add an order check function to queryit.py. I guess to support diferent types we could just make that explicit in the checks name

Reply:
This is now implemented in PR #77

Reply:
After trying to use the new order checks for this I realized that the e2e testing framework doesn't actually have the data it needs to check the ordering, as that is not selected and thus never returned by the server. As a still somewhat hackish solution I also added the aggregates to the select part of the query, with a different result variable name, which the e2e testing framework can then use to check the order.